contribution_list.md

```diff
@@ -2,5 +2,7 @@
 
 > Please add your name below to be added to the list of awesome people that have helped with this code.
 
+> Genesis Schaerrer
```

Comment:
Please update to be similar to the following.

```
- [Ciantera Rose](https://github.com/Ciantera-Rose)
- [Jacob Mason](github.com/JacobMason83)
```

Reply:
got it, will change the layout to match the existing ones on the form